matchbox_signaling/src/signaling_server/builder.rs

```diff
@@ -66,7 +66,7 @@ where
 
     /// Modify the router with a mutable closure. This is where one may apply middleware or other
     /// layers to the Router.
-    pub fn mutate_router(mut self, mut alter: impl FnMut(Router) -> Router) -> Self {
+    pub fn mutate_router(mut self, alter: impl FnOnce(Router) -> Router) -> Self {
```

Comment:
How is `FnOnce` less restrictive than `FnMut`? Not understanding, honestly.

Reply:
This allows you to use closures that can only be called once, such as ones that take ownership, which won't satisfy `FnMut`. `FnMut` by definition must satisfy `FnOnce`, so this does allow more closures/functions.